src/Symfony/Component/HttpKernel/DependencyInjection/ControllerArgumentValueResolverPass.php

```diff
@@ -40,9 +43,19 @@ public function process(ContainerBuilder $container)
             return;
         }
 
+        $resolvers = $this->findAndSortTaggedServices($this->argumentValueResolverTag, $container);
```

Comment:
assignment does not seem necessary, leftover?

Reply:
It's actually being used for `new IteratorArgument($resolvers)`. I initially wasn't able to use this as I needed the id, but casting the reference to a string gives me exactly that, saving a call to the container.